Misc. cleanup and fixes while working on discretization

pkjha-aero opened the pull request.

pkjha-aero committed:
Add comments/questions while navigating through code

ewquon committed:
Rename additional erf_* functional calls

ewquon committed:
Update reference pressure, clarify EOS functions

Based on WRF

ewquon committed:
Clarify state variable usage

Commented out conflicting and unused function declarations in header

ewquon commented on Exec/ScalarAdvection/prob.H:
```diff
@@ -49,7 +49,7 @@ pc_initdata(
   state(i, j, k, Density_comp) = ProbParm::rho_0;
  
-  // Initial temperature -- though we should make sure its consistent with (rho e) above
+  // Initial potential temperature -- though we should make sure it's consistent with (rho e) above
```
@pkjha-aero , what `rho e` are you referring to?

pkjha-aero replied:
@ewquon @jmirocha. The original comment was by Ann. I just added 'potential'. But, now that we have our equation for (rho theta), this comment about rho*e or rho*E is no longer relevant. So, we can get rid of this.

pkjha-aero replied:
These comments/edits were from days before we had our governing equations and discretizations worked out and documented.

ewquon committed:
Rename BC functions to conform to ERF naming

Comment out ERFReactFillExtDir

ewquon committed:
Update g to match WRF

ewquon committed:
Rename functions for derived quantities to conform to ERF naming

Comment out unused erf_dervel[xyz] functions that were not defined in
Derive.H. These appear to calculate velocity components from momenta

ewquon committed:
Rename additional functions/variables to conform to ERF naming

ewquon committed:
Add p_0 coefficient to EOS

pkjha-aero committed:
Update reference pressure in the documentation

pkjha-aero committed:
Update the value of g in documentation to match WRF

ewquon committed:
Remove unused declarations; update EOS comments to point to readthedocs

ewquon committed:
Remove outdated comment

ewquon commented:
@brigaudet this latest merged PR includes the ref pressure fix, FYI

brigaudet commented:
Thanks! - looks good.

--Brian


From: ewquon ***@***.***>
Sent: Thursday, April 15, 2021 6:33 PM
To: erf-model/ERF ***@***.***>
Cc: Gaudet, Brian J ***@***.***>; Mention ***@***.***>
Subject: Re: [erf-model/ERF] Misc. cleanup and fixes while working on discretization (#16)

Check twice before you click! This email originated from outside PNNL.


@brigaudet<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fbrigaudet&data=04%7C01%7Cbrian.gaudet%40pnnl.gov%7C78dd45f288484a14077408d900779afe%7Cd6faa5f90ae240338c0130048a38deeb%7C0%7C0%7C637541335967149588%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=rgQCYc5Y4h2RK%2BsBqM1lO6nOo0m5nGnS0M5zczMebqc%3D&reserved=0> this latest merged PR includes the ref pressure fix, FYI

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ferf-model%2FERF%2Fpull%2F16%23issuecomment-820844117&data=04%7C01%7Cbrian.gaudet%40pnnl.gov%7C78dd45f288484a14077408d900779afe%7Cd6faa5f90ae240338c0130048a38deeb%7C0%7C0%7C637541335967159551%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=nK%2BqgNP8%2FYmHnipm2LEHXJfRfGCQM%2B2prcwfvOivJXY%3D&reserved=0>, or unsubscribe<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FALSJSPOBGV5OB3UCO7OEAILTI6HVDANCNFSM426QI7FA&data=04%7C01%7Cbrian.gaudet%40pnnl.gov%7C78dd45f288484a14077408d900779afe%7Cd6faa5f90ae240338c0130048a38deeb%7C0%7C0%7C637541335967159551%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=wbOYangWvmic8wgslxf0Q%2Ff9B5Nd3SBsoa1ztK7kC7M%3D&reserved=0>.
